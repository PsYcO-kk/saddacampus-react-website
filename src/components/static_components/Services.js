import React, { Fragment } from 'react';

const Services = () =>(
<Fragment>
    <section className="services-area pb-120">
    <div className="container">
        <div className="row section-title">
            <h1>What Student-friendly services we offer?</h1>
            <p>BTW, we already apologize we cannot do anything about your sem-backs and low attendance, but - </p>
        </div>
        <div className="row">
            <div className="col-lg-4 single-service">
                <span className="lnr lnr-car"></span>
                <a href="#"><h4>Food Delivery</h4></a>
                <p>
                    Don't go out in the summer heat pretty. You will tan. Let us deliver right at your hostels or PG.
                </p>
            </div>
            <div className="col-lg-4 single-service">
                <span className="lnr lnr-briefcase"></span>
                <a href="#"><h4>Blogs & Informatics</h4></a>
                <p>
                    There are always some things you need to know and once in college you need all the help available.
                </p>
            </div>
            <div className="col-lg-4 single-service">
                <span className="lnr lnr-bus"></span>
                <a href="#"><h4>Deals & Offers</h4></a>
                <p>
                    Access a variety of student exclusive coupons and discounts from top online brands as well as their favourite cafes, pubs, entertainment outlets, education, hobbies and more.
                </p>
            </div>
        </div>
    </div>
    </section>
</Fragment>
)

export default Services;