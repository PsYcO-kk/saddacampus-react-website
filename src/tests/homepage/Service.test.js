import {shallow} from 'enzyme';
import React from 'react';
import Services from '../../components/homepage/Service.js';

test("testing for Service page component without props" , ()=>{
    const wrapper = shallow(<Services />)
    expect(wrapper).toMatchSnapshot();
})

test("testing for Service page component with props" , ()=>{
    const deliveryDescription = "Don't go out in the summer heat pretty. You will tan. Let us deliver right at your hostels or PG.";
    const wrapper = shallow(<Services serviceName="Food Delivery" serviceDescription={deliveryDescription} />);
    expect(wrapper).toMatchSnapshot();
})
