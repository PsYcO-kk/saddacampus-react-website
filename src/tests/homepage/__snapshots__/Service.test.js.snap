// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for Service page component with props 1`] = `
<div
  className="col-lg-4 single-service"
>
  <span
    className="lnr lnr-car"
  />
  <a
    href="#"
  >
    <h4>
      Food Delivery
    </h4>
  </a>
  <p>
    Don't go out in the summer heat pretty. You will tan. Let us deliver right at your hostels or PG.
  </p>
</div>
`;

exports[`testing for Service page component without props 1`] = `
<div
  className="col-lg-4 single-service"
>
  <span
    className="lnr lnr-car"
  />
  <a
    href="#"
  >
    <h4 />
  </a>
  <p />
</div>
`;
